core/src/main/scala/cats/Bifoldable.scala

```diff
@@ -0,0 +1,46 @@
+package cats
+
+/**
+ * A type class abstracting over types that give rise to two independent [[cats.Foldable]]s.
+ */
+trait Bifoldable[F[_, _]] extends Serializable { self =>
```

Comment:
Alright I added it for `Bifoldable`, I've added [an issue](https://github.com/typelevel/cats/issues/887) to do it for the others and I'll probably end up doing it in a future PR unless someone beats me to it